src/DocFxTocGenerator/DocFxTocGenerator.Test/output.txt

```diff
@@ -0,0 +1,12 @@
+  Determining projects to restore...
```

Comment:
this file is not needed. You should adjust where to output it

Reply:
Doh! Removed.